components/src/components/elements/included-features/IncludedFeatures.tsx

```diff
@@ -146,118 +152,160 @@ export const IncludedFeatures = forwardRef<
+                key={index}
+                ref={(el) => el && elements.current.push(el)}
+                $flexWrap="wrap"
+                $justifyContent="space-between"
+                $alignItems="center"
+                $gap="1rem"
+              >
+                <Flex $flexGrow="1" $flexBasis="min-content" $gap="1rem">
+                  {props.icons.isVisible && feature?.icon && (
+                    <IconRound
+                      name={feature.icon as IconNameTypes | string}
+                      size="sm"
+                      colors={[
+                        theme.primary,
+                        isLightBackground
+                          ? "hsla(0, 0%, 0%, 0.0625)"
+                          : "hsla(0, 0%, 100%, 0.25)",
+                      ]}
+                    />
+                  )}
 
-                {feature?.name && (
-                  <Flex $alignItems="center">
-                    <Text
-                      $font={theme.typography[props.icons.fontStyle].fontFamily}
-                      $size={theme.typography[props.icons.fontStyle].fontSize}
-                      $weight={
-                        theme.typography[props.icons.fontStyle].fontWeight
-                      }
-                      $color={theme.typography[props.icons.fontStyle].color}
+                  {feature?.name && (
+                    <Flex $alignItems="left" $flexDirection="column">
+                      <Text
+                        $font={
+                          theme.typography[props.icons.fontStyle].fontFamily
+                        }
+                        $size={theme.typography[props.icons.fontStyle].fontSize}
+                        $weight={
+                          theme.typography[props.icons.fontStyle].fontWeight
+                        }
+                        $color={theme.typography[props.icons.fontStyle].color}
+                      >
+                        {feature.name}
+                      </Text>
+                      <Text
+                        $font={
+                          theme.typography[
+                            props.entitlementExpiration.fontStyle
+                          ].fontFamily
+                        }
+                        $size={
+                          theme.typography[
+                            props.entitlementExpiration.fontStyle
+                          ].fontSize
+                        }
+                        $weight={
+                          theme.typography[
+                            props.entitlementExpiration.fontStyle
+                          ].fontWeight
+                        }
+                        $leading={1}
+                        $color={
+                          theme.typography[
+                            props.entitlementExpiration.fontStyle
+                          ].color
+                        }
+                      >
+                        {entitlementExpirationDate && (
```

Comment:
We should move this "truthy" check up one level so that an empty `Text` component (span element) doesn't get rendered if there is no `entitlementExpirationDate`.

```tsx
{entitlementExpirationDate && (
  <Text
    ...
  >
    ...
  </Text>
)}
```

Reply:
Fixed